modules/apps/27-interchain-accounts/controller/keeper/msg_server.go

```diff
@@ -39,7 +40,12 @@ func (s msgServer) RegisterInterchainAccount(goCtx context.Context, msg *types.M
 
 	s.SetMiddlewareDisabled(ctx, portID, msg.ConnectionId)
 
-	channelID, err := s.registerInterchainAccount(ctx, msg.ConnectionId, portID, msg.Version, msg.Order)
+	// use ORDER_ORDERED as default in case msg's ordering is NONE
+	order := channeltypes.ORDERED
+	if msg.Order == channeltypes.UNORDERED {
+		order = channeltypes.UNORDERED
+	}
```

Comment:
I think I prefer leaving the msg unchained TBH, but I won't fight anyone over it :sweat_smile:

Reply:
I think its slightly confusing the way it's written? I'd prefer Damian's suggestion tbh, feels more explicit to me

Reply:
OK, I took the Solomonic decision of not mutating the message (since I also like that) but making the code more explicit (at the cost of more lines of code, unfortunately). I hope that makes everyone (reasonably) happy. :)